### PR TITLE
SWPWA-Share wishlist button

### PR DESCRIPTION
**Pull request by AleksandrsKondratjevs**

### CODE REVIEW

File: src/app/component/Header/Header.component.js [hunk 1]
```diff
@@ -33,6 +33,7 @@ import CSS from 'Util/CSS';
 import media from 'Util/Media';
 import { LOGO_MEDIA } from 'Util/Media/Media';
 
+import wishListShareButton from '../../../public/assets/images/global/shareWishList.svg';
```
Comment (alfredsgenkins) [inline src/app/component/Header/Header.component.js:36]: Can you please import SVG as an asset, not from public? :D

Comment (alfredsgenkins) [reply src/app/component/Header/Header.component.js:36]: Public folder is for files used in index.html / .phtml

**Review by alfredsgenkins — CHANGES_REQUESTED** [review]
It lloks good except this import.

File: src/app/component/Header/Header.component.js (hunk 1 above)
Comment (alfredsgenkins) [reply src/app/component/Header/Header.component.js:36]: Maybe we have a similar icon in `_icons.scss`?

Comment (AleksandrsKondratjevs) [reply src/app/component/Header/Header.component.js]: arrow back + minicart can be a solution.

File: src/app/style/abstract/_icons.scss [hunk 2]
```diff
@@ -473,3 +473,66 @@
     margin: 0 4px;
     width: 20px;
 }
+
+@mixin share-button {
+    $share-button-stroke-width: calc(var(--header-icon-stroke-width) + 1px);
```
Comment (AleksandrsKondratjevs) [inline src/app/style/abstract/_icons.scss:478]: Share button is not getting value on mobile from navigationTabs component (2px) and on desktop it should be 2px also.

**Review by yeegor — APPROVED** [review]
Seems legit

**Pull request merged**